src/BTVNanoCommissioning/utils/compile_jec.py

```diff
@@ -298,6 +298,14 @@ def jet_factories(campaign):
                     "src/BTVNanoCommissioning/data/JME/Winter22Run3/Winter22Run3_RunC_V1_DATA_L2L3Residual_AK4PFPuppi.jec.txt",
                 ]
             ),
+            "dataD": jet_factory_factory(
```

Comment:
Could you rename the JEC files in `dataC` from `V1` to `V2`? So that the naming scheme is consistent? Renaming happened in [this commit](https://github.com/anmalara/JECDatabase/commit/6e55c1192b19d969eff785e58d19d592ceff9440) by JetMET.